examples/experimental/dagster-airlift/dagster_airlift/core/serialization/compute.py

```diff
@@ -62,10 +74,20 @@ def asset_key_map(self) -> Dict[str, Dict[str, Set[AssetKey]]]:
                     asset_key_map[task_handle.dag_id][task_handle.task_id].add(spec.key)
         return asset_key_map
 
+    @cached_property
+    def asset_keys_by_dag(self) -> Dict[str, Set[AssetKey]]:
+        """Mapping of dag_id to set of asset_keys mapped from that dag."""
+        asset_key_map: Dict[str, Set[AssetKey]] = defaultdict(set)
+        for spec in self.asset_specs:
+            if is_dag_mapped_asset_spec(spec):
+                for dag_handle in dag_handles_for_spec(spec):
+                    asset_key_map[dag_handle.dag_id].add(spec.key)
+        return asset_key_map
+
```

Comment:
this naming is confusing alongside `asset_keys_in_dag_by_id` IMO. `asset_keys_by_dag` should have a more specific name to indicate only contains DAG to asset mappings

Reply:
`asset_keys_by_mapped_dag_id`?